src/api/routes/sats.ts

```diff
@@ -37,7 +37,13 @@ export const SatRoutes: FastifyPluginCallback<Record<never, never>, Server, Type
       },
     },
     async (request, reply) => {
-      const sat = new OrdinalSatoshi(request.params.ordinal);
+      let sat: OrdinalSatoshi;
+      try {
+        sat = new OrdinalSatoshi(request.params.ordinal);
+      } catch (error) {
+        await reply.code(400).send({ error: 'Not found' });
```

Comment:
Could be a different http code or message, might be confusing to see Not found on non-404. Or could even use the error message from the throw

Reply:
Done, good point.